docs/UserGuide.md

```diff
@@ -122,26 +122,26 @@ The following is an annotated breakdown of **MODTrek's** GUI:
 messages and results.
 3. **Results Section:** This panel shows your degree progression by default. This panel can be changed using the `view` command
 to toggle between the module list in **MODTrek** or the user's degree progression. Using the `find` command when making 
-a query on the module list will navigate you to the module search screen.
+a query on the module list will navigate you to the module search screen.<br><br><br><br>
 
-The following is the annotated breakdown of the Results Section of **MODTrek's** GUI:
-![UI subsection](images/Ui-subsections-detailed.png)
-<p align="center"><em>Breakdown of Results Section</em><br></p>
+The following shows the various screens of the Results Section in **MODTrek's** GUI:
+![UI subsection](images/Ui-Dashboard.png)
+<p align="center"><em>Screens of Results Section</em><br></p>
 
-1. **Doughnut Chart:** This will be shown in the degree progress section. It shows your completion progress of the degree
-and individual sub requirements of the degree. Your total MCs completed and CAP are also displayed.
-2. **Module List:** This will be shown in both the module list section and module search section. In the module list section, all the modules in the list are shown. In the module search section, all the modules in the list are shown by default. If a filter is applied by the find command, only filtered modules are shown.
-3. **Module Group:** This will be shown in both the module list section and module search section. In the module list section, the modules in each group
-belonging to a sub category are shown (sorted by year-sem by default). In the module search section, all the relevant modules are shown as a single group.
-4. **Module Card:** This will be shown in both the module list section and module search section. Each module card shows the detail of each module.
+1. **Degree Progress Screen:** This screen shows the completion progress of the degree and individual sub requirements of the 
+degree. Your total MCS completed and CAP are also displayed in a doughnut chart.
+2. **Module List Screen:** This screen shows all your current modules tracked by MODTrek. Here, you can customise the 
+way you want to categorise your modules. Refer to the section on [Sort all modules](#sort-module) for more details!
+3. **Module Search Screen:** This screen shows your filtered list of modules. You can customise the filters to search
```

Comment:
I think these are better names and descriptions of the different interfaces compared to before.